uui-core/types/tables.ts

```diff
@@ -308,16 +308,18 @@ export type IFilterPredicate = {
 type FilterConfigBase<TFilter> = {
     title: string;
     field: keyof TFilter;
-    columnKey?: string;
+    columnKey: string;
     isAlwaysVisible?: boolean;
-    predicates?: IFilterPredicate[]
+    predicates?: IFilterPredicate[];
+    format?: string;
```

Comment:
move 'format' into `DatePickerFilterConfig` because it should be used only for date pickers